src/codeu/chat/client/core/View.java

```diff
@@ -162,3 +162,19 @@ public ServerInfo getInfo() {
     return null;
 }
 }
+
+  public ServerInfo getInfo() {
```

Comment:
You can delete this section as well. Your merge brought in a duplicated version of this code from master. (See lines 146-164)